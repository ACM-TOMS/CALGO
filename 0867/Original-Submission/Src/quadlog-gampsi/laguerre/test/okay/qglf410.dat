### Numerical integration with qglqf()
###
### int(sin(sigma*x) * x^alpha * exp(-x) * ln(x), x = 0..infinity)
###
### Line format: 2 sigma alpha neval result relerr abserr
###
2    -1.000000000000000000E+00   1.000000000000000000E+00         50   -3.810537240921558863E-02  8.31E-13 -3.17E-14
2     1.000000000000000000E+00   0.000000000000000000E+00         50   -6.919554589202915692E-02 -8.01E-15  5.54E-16
