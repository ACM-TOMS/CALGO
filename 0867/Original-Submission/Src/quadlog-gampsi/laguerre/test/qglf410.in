25 -1 1
0 1 1
1 1 -2
25 1 0
32767 1 1
