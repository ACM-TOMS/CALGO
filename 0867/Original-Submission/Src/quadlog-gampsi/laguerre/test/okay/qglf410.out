  Test integral: int(sin(sigma*x) * x^alpha * exp(-x) * ln(x), x = 0..infinity)

  1 ulp =   1.93E-34

  Floating-point precision appears to be (1 - log2(ulp) =) 113 bits

   nquad     sigma     alpha     Quadrature Result        Exact Integral     Rel. Error RelE (ULPs)  Err (bits)
      25   -1.0000    1.0000  -3.810537240921559E-02  -3.810537240924724E-02   8.31E-13    4.31E+21     71.87
      25    1.0000    0.0000  -6.919554589202916E-02  -6.919554589202860E-02  -8.01E-15   -4.16E+19     65.17

  Maximum relative error =   8.31E-13   4.31E+21 ulps      71.87 bits at nquad =   25 sigma =   -1.0000 alpha =    1.0000
  Minimum relative error =  -8.01E-15  -4.16E+19 ulps      65.17 bits at nquad =   25 sigma =    1.0000 alpha =    0.0000
  Average relative error =   4.19E-13   2.18E+21 ulps      70.88 bits

  Done
